### DISCUSSION TITLE
Which repository should be used for issues and PRs related to mip?

### DISCUSSION THREAD

**Discussion by ned-pcs** [micropython/micropython#11767] [General]
It's unclear to me which repository should be used for issues and PRs related to `mip` since there's one copy of the code at `micropython-lib/micropython/mip/__init__.py` and one at `micropython/tools/mpremote/mpremote/mip.py`.

I've issued PRs and issues in `micropython-lib` but have seen others in `micropython`.

@andrewleech @jimmo what do you think?

**Comment by andrewleech** [comment]
In terms of general mip issues, I'd expect the main repo to be the better location unless it's pointing to particular pieces of the -lib mip code.

A PR would need to go wherever the code it's targeting lives, either the mpremote version in the main library or the mpy version in -lib. 

This is just my assumption though.

**Reply by ned-pcs** [reply]
The problem is that the main repo has a (renamed) copy of the one in -lib.

**Comment by jimmo** [comment]
@ned-pcs As we need to keep both in sync, then any behavioral/functional change to `mip` would need to be a PR to both repos.

That said, the "primary" version is the one in `micropython-lib`, so issues are best raised there. There's only one discussions forum shared for both repos.

Unfortunately the one in the main repo isn't just a rename, the implementation is slightly different.

**Reply by jimmo** [reply]
(Btw I will try and take a look at https://github.com/micropython/micropython-lib/pull/676 shortly)
